olca-app/src/org/openlca/app/devtools/csharp/CSharp.java

```diff
@@ -0,0 +1,116 @@
+package org.openlca.app.devtools.csharp;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.util.Objects;
+import java.util.Properties;
+
+import org.apache.commons.io.FileUtils;
+import org.openlca.app.App;
+import org.openlca.app.Config;
+import org.openlca.app.db.Database;/*MB Additional import*/
+import org.openlca.app.db.IDatabaseConfiguration;/*MB Additional import*/
+import org.openlca.app.devtools.ScriptApi;
+import org.openlca.app.devtools.csharp.server.Server;
+import org.openlca.app.navigation.actions.DatabaseActivateAction;/*MB Additional import*/
+import org.openlca.app.rcp.RcpActivator;
+import org.openlca.core.database.IDatabase;/*MB Additional import*/
+import org.python.util.PythonInterpreter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.zeroturnaround.zip.ZipUtil;
+
+
+class CSharp {
+
+	private static IDatabase database;
+	
+	private static Logger log = LoggerFactory.getLogger(CSharp.class);
+	
+	private static boolean initialized = false;
+
+	public static void eval(String script) {
+		try {
+			if (!initialized)
+				initialize();
+			String fullScript = prependImports(script);
+			doEval(fullScript);
+		} catch (Exception e) {
+			/*Logger log = LoggerFactory.getLogger(CSharp.class);*/
+			log.error("failed to evaluate script", e);
+		}
+	}
+
+	private static void doEval(String script) {
+		try (PythonInterpreter interpreter = new PythonInterpreter()) {
```

Comment:
So the idea is to run a Python script which can interact with a C# backend over sockets?